solidstart/github-oauth/src/routes/login/github/callback.ts

```diff
@@ -49,8 +43,10 @@ export async function GET(event: APIEvent) {
 			githubUser.login
 		);
 		const session = await lucia.createSession(userId, {});
-		appendHeader(event, "Set-Cookie", lucia.createSessionCookie(session.id).serialize());
-		return sendRedirect(event, "/");
+		const cookie = lucia.createSessionCookie(session.id);
+
+		setCookie(cookie.name, cookie.value, cookie.attributes);
+		return Response.redirect("/");
```

Comment:
This is throwing a type error because it can't parse the URL. Instead you should probably use [redirect](https://docs.solidjs.com/solid-router/reference/data-apis/response-helpers#redirect) from `@solidjs/router`